### ISSUE TITLE
No error message shown when adding a contact using a disable protocol 

### ISSUE THREAD

**Issue by DetroitLibertyPenguin** [friendica/friendica#11808]
- [ x] I have searched open and closed issues for duplicates

### Steps to Reproduce

1.  step one pump.io is disabled on instance 
2.  step two type https://identi.ca/debian as a dress of contact to ass
3.  step three click folliw

Actual Result:
No error message given, user does not show in list of contacts. 

Expected Result:some message indicating network protocol pump.io disabled on server side

### Screenshots

https://libranet.de/display/0b6b25a8-5762-de4f-d0d1-44d460740425

### Platform Info

Friendica Version:

This is Friendica, version 2022.09-dev that is running at the web location https://libranet.de. The database version is 1475/1475, the post update version is 1452/1452.

**Comment by Quix0r** [comment]
I cannot reproduce it with latest `develop` code. I was able to connect to the pump.io account with no error messages. But I'm not sure if my account is really following it.
https://cloud.mxchange.org/index.php/s/public?path=%2FFriendica%2F202208

**Comment by DetroitLibertyPenguin** [comment]
That sounds like what I’m discribing, please be aware that the admin of the instance I’m on indicated that he disabled Pump.io support from the back end as do few people still use it he might as well save some resource power on his server. That’s why it’s particularly surprising I’m getting the results I did.


Sent from my Smartphone, please dismiss not-so-smart typos.
________________________________

**Comment by AlfredSK** [comment]
I'm not sure. I always thought that pump.io is supported via the pumpio add-on. Have we changed that to a natively supported protocol?

**Comment by annando** [comment]
No, we don't natively support pump.io. But some time ago the probing for profiles had been updated to detect pump.io profiles.

**Comment by DetroitLibertyPenguin** [comment]
@annando so I'm correct that I should have been able to see it when I attempt to add it, but if my instance doesn't support the add-on I should have recieved some failure message indicating why I could find it but not add it?

**Comment by annando** [comment]
I'm unsure about that.

**Comment by AlfredSK** [comment]
To mitigate the issue on libranet.de I have now enabled the pumpio add-on. :-)

**Comment by DetroitLibertyPenguin** [comment]
Just in time for identi.ca to have been offline. The question is, is it forever?

Sent from my Smartphone please dismiss no-so-smart-typos
________________________________

**Comment by DetroitLibertyPenguin** [comment]
> To mitigate the issue on libranet.de I have now enabled the pumpio add-on. :-)
Doesn't seem to actually improved anything. I don't see the inconsistent error message anymore, but it still looks like nothing is getting updated. 

For example https://datamost.com/rodross has had several updates recently, but I see nothing in my feed. [https://libranet.de/photo/6884307021632a21715ce30654145739-0.png](url)

**Comment by DetroitLibertyPenguin** [comment]
> I'm unsure about that.



> @annando so I'm correct that I should have been able to see it when I attempt to add it, but if my instance doesn't support the add-on I should have recieved some failure message indicating why I could find it but not add it?

@annando perhaps I should say I would expect to see a failure message indicating pump.io isn't supported by my instance. Whether such a feature-request is worth it is another questoin
